src/strategies/security-wg.js

```diff
@@ -13,6 +14,11 @@ import * as cache from "../cache.js";
 import { standardizeVulnsPayload } from "./vuln-payload/standardize.js";
 
 export async function SecurityWGStrategy(options = {}) {
+  emitWarning('Node.js Security WG DB is deprecated and will be removed soon.', {
```

Comment:
It would prefer if you use `process.emitWarning` in one shot (no need to destruct it at the top).